fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -301,23 +282,27 @@ public static void init() {
                 if (includedSPs.containsKey(code)) {
                     String configuredUrl = includedSPs.get(code);
                     if (configuredUrl != null && configuredUrl.equals(url)) {
-                        results.add(sp);
-                    } else {
+                        results.put(code, sp);
+                    } else if (log.isLoggable(Level.FINE)) {
                         log.info("Skipping search parameter with id='" + sp.getId() + "'. "
```

Comment:
Do you want to use log.fine here, since that's what you're checking against in the line above?

Reply:
```suggestion
                        log.fine("Skipping search parameter with id='" + sp.getId() + "'. "
```